core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -20,30 +20,48 @@ package org.apache.spark.deploy.master
 import akka.actor.Address
 import org.scalatest.FunSuite
 
-import org.apache.spark.SparkException
+import org.apache.spark.{SparkConf, SparkException}
 
 class MasterSuite extends FunSuite {
 
   test("toAkkaUrl") {
-    val akkaUrl = Master.toAkkaUrl("spark://1.2.3.4:1234")
+    val conf = new SparkConf(loadDefaults = false)
```

Comment:
`loadDefaults = false` is a nice touch; we should probably do this in more of our unit tests.